Assets/Scripts/Statistics/BasicStatistics/Dexterity.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dexterity : Statistic
+{
+    public Dexterity(float baseValue, float currentValue)
```

Comment:
Jeśli baseValue i currentValue mają zawsze tą samą wartość przy inicjalizacji -> a tak widziałem, to możesz jedną wartość mieć w konstruktorze, która ustawi base i current value.